tests/pipeline-all-amps/repo/dtas/dtas-config.json

```diff
@@ -3,8 +3,7 @@
         "host": "http://localhost:8080",
         "username": "admin.pipeline@alfresco.com",
         "password": "admin",
-        "request_timeout_seconds": "45",
-        "wait_rendition_max_attempts": "15"
```

Comment:
`wait_rendition_max_attempts` has been removed from DTAS and replaced with other properties as per https://github.com/Alfresco/alfresco-deployment-test-automation-scripts/pull/138/files. Given the tests pass without changing the default values, I decided to remove the deprecated property and not override the new ones.